### PR TITLE
Support for ActivePython

### PR DESCRIPTION
**Pull request by timsavage**
Support for installing the ActivePython binary distribution for Linux. 

Includes the current releases.

### CODE REVIEW

File: plugins/python-build/share/python-build/activepython-2.7.14 [hunk 1]
```diff
@@ -0,0 +1,13 @@
+case "$(activepython_architecture 2>/dev/null || true)" in
+"linux-x86_64" )
+  install_package "ActivePython-2.7.14.2717-linux-x86_64-glibc-2.12-404899" "http://downloads.activestate.com/ActivePython/releases/2.7.14.2717/ActivePython-2.7.14.2717-linux-x86_64-glibc-2.12-404899.tar.gz" "activepython" verify_py27
```
Comment (yyuu) [inline plugins/python-build/share/python-build/activepython-2.7.14]: Could you add SHA256 checksum of this archive like other definitions...?

https://github.com/pyenv/pyenv/blob/v1.2.2/plugins/python-build/share/python-build/3.6.4#L5

File: plugins/python-build/share/python-build/activepython-3.5.4 [hunk 2]
```diff
@@ -0,0 +1,13 @@
+case "$(activepython_architecture 2>/dev/null || true)" in
+"linux-x86_64" )
+  install_package "ActivePython-3.5.4.3504-linux-x86_64-glibc-2.12-404899" "http://downloads.activestate.com/ActivePython/releases/3.5.4.3504/ActivePython-3.5.4.3504-linux-x86_64-glibc-2.12-404899.tar.gz" "activepython" verify_py35
```
Comment (yyuu) [inline plugins/python-build/share/python-build/activepython-3.5.4]: ditto

File: plugins/python-build/share/python-build/activepython-3.6.0 [hunk 3]
```diff
@@ -0,0 +1,16 @@
+case "$(activepython_architecture 2>/dev/null || true)" in
+"linux-x86_64" )
+  install_package "ActivePython-3.6.0.3600-linux-x86_64-glibc-2.3.6-401834" "http://downloads.activestate.com/ActivePython/releases/3.6.0.3600/ActivePython-3.6.0.3600-linux-x86_64-glibc-2.3.6-401834.tar.gz" "activepython" verify_py36
```
Comment (yyuu) [inline plugins/python-build/share/python-build/activepython-3.6.0]: ditto

File: plugins/python-build/share/python-build/activepython-3.6.0 [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  ;;
+"linux-x86" )
+  install_package "ActivePython-3.6.0.3600-linux-x86-glibc-2.3.6-401834" "http://downloads.activestate.com/ActivePython/releases/3.6.0.3600/ActivePython-3.6.0.3600-linux-x86-glibc-2.3.6-401834.tar.gz" "activepython" verify_py36
```
Comment (yyuu) [inline plugins/python-build/share/python-build/activepython-3.6.0]: ditto

File: plugins/python-build/share/python-build/activepython-2.7.14 [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+  ;;
+* )
+  { echo
+    colorize 1 "ERROR"
+    echo ": The binary distribution of Active Python is not available for $(activepython_architecture 2>/dev/null || true)."
```
Comment (joshfriend) [inline plugins/python-build/share/python-build/activepython-2.7.14]: they have distributions for macOS, x86 linux and windows as well

Comment (timsavage) [reply plugins/python-build/share/python-build/activepython-2.7.14]: I'm aware of the Windows and OSX distributions, however, I don't use either of those platforms and am not able to test them. 

X86 is included, only Python 3.6 includes an x86 package.

**Pull request merged**